updates to .js file: added functions to work with APIs

kZhaanbaev opened the pull request:
added functions:

1. getSattelitesNearMe(lat, lng, alt, searchRadius, categoryID)
2. getUserCoordinatesFromCityName(cityName)

added objects:

1. satteliteCategories - list of sattelite categories

kZhaanbaev committed:
fixed link issue in README.md file, was broken for some reason

kZhaanbaev committed:
added a function that gets all sattelites based on user's location

kZhaanbaev committed:
added a function getUserCoordinatesFromCityName(cityName), object satteliteCategories that contains all sattelite categories

Captain63 commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```
Looks like the live server configurations were tracked during the Add stage? I'll update gitignore with the next pull request

Captain63 committed:
Deleting liveServer configuration file

Looks like the liveServer configuration file was added to tracked files during an Add. No need to store on the repo

Captain63 reviewed (approved):
The API calls you have set up look great! The comments make them easy to follow.
